.github/actions/install-win-store-python/action.yml

```diff
@@ -0,0 +1,51 @@
+name: Install Windows Store Python
+description: Installs Python from the Windows Store
+
+inputs:
+  python-version:
+    description: 'Python version to install'
+    default: '3.11'  # should default to the latest Python
+    required: false
+
+runs:
+  using: composite
+  steps:
+
+    - name: Python Constants
+      id: python
+      shell: powershell
+      run: |
+        # Constants for all Python versions
+        echo "exe-dir=$(Join-Path -Path $env:LOCALAPPDATA -ChildPath \Microsoft\WindowsApps\)" | Out-File -FilePath $Env:GITHUB_OUTPUT -Encoding utf8 -Append
+
+        # Constants for each Python version
+        if ( "${{ inputs.python-version }}" -eq "3.11" ) {
```

Comment:
Presumably we could/should fill this out for every other version we want to support, providing a download URL?

Reply:
Added back to Python 3.8. Added notes about the....usability of these Pythons before 3.10...